server/database/quieres/teacherProfile.js

```diff
@@ -0,0 +1,7 @@
+const connection = require('../config/connection');
+
+const getTeacherProfile = () => connection.query(
+  'select * from teacher inner join "user" on teacher.user_id="user".id ',
```

Comment:
selecting everything from the user table with return the users' passwords! that's a serious security flaw, despite the fact the passwords are hashed :) I'll create an issue for this, but here you can just select only the columns you actually need